### ISSUE TITLE
Is every user expected to maintain a complete data record of every website they visit??

### ISSUE THREAD

**Issue by didlie** [HelloZeroNet/ZeroNet#1335]
How can this work? How can this be possible? Every peer is expected to serve the entire Internet?... at least the entire Znet-U-verse??

### Step 1: Please describe your environment

  * ZeroNet version: _____
  * Operating system: _____
  * Web browser: _____
  * Tor status: not available/always/disabled
  * Opened port: yes/no
  * Special configuration: ____

#### Steps to reproduce:

  1. _____
  2. _____
  3. _____

#### Observed Results:

  * What happened? This could be a screenshot, a description, log output (you can send log/debug.log file to hello@zeronet.io if necessary), etc.

#### Expected Results:

  * What did you expect to happen?

**Comment by skwerlman** [comment]
no, they are only expected to maintain each site they visit, along with any optional files they choose to use. of course, its possible to delete a downloaded site if the user doesn't want to serve it any longer.

**Comment by didlie** [comment]
I noticed that deleted site files can update and override the delete and reappear after being deleted. I deleted about 5 sites sequentially and only the first one deleted - maybe the second one also. I went to the /data/ and deleted another site and it also reappeared when I restarted zeronet. Am I just not giving the program enough time to complete the delete? or is it possible that some other method is rewriting the local site data?
